docs/usage/node.md

```diff
@@ -19,6 +19,7 @@ Renovate can manage the Node.js version in the following files:
 - The [`engines`](https://docs.npmjs.com/files/package.json#engines) field in [`package.json`](https://docs.npmjs.com/files/package.json)
 - The [`volta`](https://docs.volta.sh/guide/understanding#managing-your-project) field in [`package.json`](https://docs.npmjs.com/files/package.json)
 - The [`.nvmrc`](https://github.com/creationix/nvm#nvmrc) file for the [Node Version Manager](https://github.com/creationix/nvm)
+- The [`.node-version`](https://github.com/nodenv/nodenv#choosing-the-node-version) file for the [Nodenv](https://github.com/nodenv/nodenv) environment manager
```

Comment:
```suggestion
- The [`.node-version`](https://github.com/nodenv/nodenv#choosing-the-node-version) file for the [nodenv](https://github.com/nodenv/nodenv) environment manager
```

The project itself doesn't capitalize their name, so we should follow their example.